pacsman/filesystem_dev_client.py

```diff
@@ -127,6 +127,21 @@ def fetch_thumbnail(self, series_id):
         for (path, ds) in dicom_datasets.items():
             if ds.SeriesInstanceUID == series_id:
                 series_paths.append(path)
-        # TODO check sorting of IDs
-        if series_paths:
-            shutil.copy(series_paths[len(series_paths) // 2], self.dicom_dir)
+        if not series_paths:
+            return None
+
+        thumbnail_series_path = series_paths[len(series_paths) // 2]
```

Comment:
How do we know this will be the middle slice?  Are the slices ordered?

Reply:
With that big phantom dataset, Horos orders by image instance ID. I can't find anything that requires slices to be in instance ID order, so it could just be that it's returning in slice order (i.e. based on `ImagePositionPatient` / `SlicePosition`) and it happens to line up. I think sorting by instance ID is the right thing here. Right now it's just using a `dict` order.